src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeFindingPositiveRiskCellConfigurator.swift

```diff
@@ -0,0 +1,82 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors /
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import UIKit
+
+final class HomeFindingPositiveRiskCellConfigurator: HomeRiskCellConfigurator {
+
+	let identifier = UUID()
+
+	var nextAction: (() -> Void)?
+
+	// MARK: Configuration
+
+	func configure(cell: RiskFindingPositiveCollectionViewCell) {
+
+		cell.delegate = self
+
+		cell.removeAllArrangedSubviews()
```

Comment:
would this fit better in prepareForReuse() of the cell?

Reply:
good point, fixed